### ISSUE TITLE
use a smaller LR?

### ISSUE THREAD

**Issue by 152334H** [tloen/alpaca-lora#4]
The Karparthy Constant used currently might be too high? The loss for this training run is not going down after LR increases beyond ~1e-4:

![image](https://user-images.githubusercontent.com/54623771/224908966-fd260c1e-82da-4399-9ac0-da8594779a7e.png)

**Comment by tloen** [comment]
Updating to 2e-5 as in the original paper. I'll check in the morning if it works better.

**Comment by 152334H** [comment]
![image](https://user-images.githubusercontent.com/54623771/224926271-4242bca1-7c4c-4645-969d-84a7e09f2c42.png)

this doesn't seem to actually help. Purple line is with 2e-5, green line 3e-4.

Maybe this is just the best result obtainable with the q_proj/v_proj parameters?

**Comment by tloen** [comment]
Have you evaluated the model quality? I've always suspected that instruct-tuning is much less data-intensive than most people think.

**Comment by tloen** [comment]
Could also be worth trying to disable the int8 quantization. Will check tomorrow.

**Comment by kesar** [comment]
<img width="1340" alt="Captura de pantalla 2023-03-15 a las 22 49 47" src="https://user-images.githubusercontent.com/1288106/225452037-f8810d8e-6b3e-476c-ad31-34345d3ef555.png">

same here.

only changed:
BATCH_SIZE = 256
MICRO_BATCH_SIZE = 5

**Comment by tloen** [comment]
Fwiw I've been able to eke out some small gains setting LORA_R to 8 instead of 4. Otherwise, seems (until proven otherwise) like both learning rates are perfectly fine.
